policy/README.md

```diff
@@ -289,7 +289,7 @@ An individual `Rule` object is defined by the following fields:
 | `geography_id`   | UUID      | Required   | Unique ID of [Geography](/geography#general-information)                                               |
 | `geography_json`   | UUID      | Required   | The GeoJSON that defines the geographical coordinates.
 | `effective_date`   | [timestamp][ts] | Optional   | `start_date` for first published policy that uses this geo.  Server should set this when policies are published.  This may be used on the client to distinguish between “logical” geographies that have the same name. E.g. if a policy publishes a geography on 5/1/2020, and then another policy is published which references that same geography is published on 4/1/2020, the effective_date will be set to 4/1/2020.
-| `publish_date`   | [timestamp][ts] | Required   | Timestamp that the policy was published, i.e. made immutable                                             |
```

Comment:
This is already 'published_date' in Policy in dev: https://github.com/openmobilityfoundation/mobility-data-specification/blob/dev/policy/README.md#policy

Reply:
Sorry, not sure I understand you comment. I changed the part under "Geography" here: https://github.com/openmobilityfoundation/mobility-data-specification/blob/dev/policy/README.md#geography

Reply:
Oh I see you changed the geography field currently within Policy API, not the field in the policy endpoint. Makes sense.